### PR TITLE
Proofread Metadata section

### PR DESCRIPTION
**Pull request by selfissued**
Correct issues identified while proofreading the Metadata section.

### CODE REVIEW

**Review by peppelinux — APPROVED** [review]

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 1]
```diff
@@ -1282,19 +1282,19 @@ Cache-Control: no-store
...
 
-How to obtain Client Metadata is out of scope of this specification. The profiles of this specification may define static Client Metadata values.
+How to obtain Client Metadata is out of scope of this specification. Profiles of this specification MAY also define static sets of Client Metadata values to be used.
 
 If the Credential Issuer is unable to perform discovery of the Wallet's Credential Offer Endpoint, the following claimed URL is used: `openid-credential-offer://`.
 
 ## Credential Issuer Metadata {#credential_issuer_metadata}
 
-The Credential Issuer Metadata contains information on the Credential Issuer's technical capabilities, supported Credentials and (internationalized) display information.
+The Credential Issuer Metadata contains information on the Credential Issuer's capabilities, supported Credentials, and (internationalized) display information.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1297]: I think `technical` is important here. it's not like any capability of the issuer can be obtained from the metadata

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 2]
```diff
@@ -1327,52 +1327,52 @@ Accept-Language: fr-ch, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
...
-* `require_credential_response_encryption`: OPTIONAL. Boolean value specifying whether the Credential Issuer requires additional encryption on top of TLS for the Credential Response and expects encryption parameters to be present in the Credential Request and/or Batch Credential Request, with `true` indicating support. When the value is `true`, `credential_response_encryption_alg_values_supported` parameter MUST also be provided.  If omitted, the default value is `false`.
+* `require_credential_response_encryption`: OPTIONAL. Boolean value specifying whether the Credential Issuer requires additional encryption on top of TLS for the Credential Response and expects encryption parameters to be present in the Credential Request and/or Batch Credential Request, with `true` indicating support. When the value is `true`, the `credential_response_encryption_alg_values_supported` and `credential_response_encryption_enc_values_supported` parameters MUST also be provided.  If omitted, the default value is `false`.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1340]: for this part we need to wait for PR #153

File: openid-4-verifiable-credential-issuance-1_0.md [hunk 3]
```diff
@@ -1327,52 +1327,52 @@ Accept-Language: fr-ch, fr;q=0.9, en;q=0.8, de;q=0.7, *;q=0.5
...
+  * `cryptographic_binding_methods_supported`: OPTIONAL. Array of case sensitive strings that identify how the Credential is bound to the identifier of the End-User who possesses the Credential, as defined in (#credential-binding). Support for keys in JWK format [@!RFC7517] is indicated by the value `jwk`. Support for keys expressed as a COSE Key object [@!RFC8152] (for example, used in [@!ISO.18013-5]) is indicated by the value `cose_key`. When the Cryptographic Binding Method is a DID, valid values are a `did:` prefix followed by a method-name using a syntax defined in Section 3.1 of [@!DID-Core], but without a `:`and method-specific-id. For example, support for the DID method with a method-name "example" would be represented by `did:example`. Support for all DID methods listed in Section 13 of [@DID_Specification_Registries] is indicated by sending a DID without any method-name.
```
Comment (Sakurann) [inline openid-4-verifiable-credential-issuance-1_0.md:1348]: this is not editorial if it remove a `MUST`

**Review by Sakurann — DISMISSED** [review]
I said it in PR #179 too, but would prefer to keep the abbreviation "AS" for the authorization server.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 1 above)
Comment (selfissued) [inline openid-4-verifiable-credential-issuance-1_0.md:1297]:
```suggestion
The Credential Issuer Metadata contains information on the Credential Issuer's technical capabilities, supported Credentials, and (internationalized) display information.
```

Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md]: I restored "technical".

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 3 above)
Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md:1348]: The new wording has the same meaning as the prior, but reads more naturally.  "is" is just as normative as "MUST".  Declarative sentences not using RFC 2119 keywords are just as normative as those using them.  It's editorial (and improves the readability).

**Comment by selfissued** [comment]
@jogu's comment at https://github.com/openid/OpenID4VCI/pull/179#issuecomment-1877190518 also applies here:
> I'd personally suggest using "Authorization Server". I don't think we define the term "AS" anywhere and even if we did define it, I think it makes the specifications hard to understand for people new to the space.

File: openid-4-verifiable-credential-issuance-1_0.md (hunk 2 above)
Comment (bc-pi) [reply openid-4-verifiable-credential-issuance-1_0.md:1340]: #153 has been merged and yeah now there are conflicts that need to be resolved :(

Comment (selfissued) [reply openid-4-verifiable-credential-issuance-1_0.md]: Merge conflicts resolved

**Review by bc-pi — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Review by awoie — APPROVED** [review]

**Pull request merged**